cmd/new/new.go

```diff
@@ -25,9 +25,10 @@ func Run(content string) error {
 	return nil
 }
 
-func makeNote(content string) utils.Note {
-	return utils.Note{
-		UID:     utils.GenerateNoteID(),
+func makeNote(notename string, content string) utils.Note {
+	return utils.Note {
+		UID: utils.GenerateNoteID(),
+		Name: notename,
```

Comment:
Maybe a small check to see if `notename` isn't empty. Otherwise, it might be useful to populate it with the result of a call to `GenerateNoteName`.